nixcon-2022.md

```diff
@@ -0,0 +1,19 @@
+
+The talk that was most CHAINS-adjacent was without doubt [_Reproducibly building artifacts that contain embedded signatures_](https://talks.nixcon.org/nixcon-2022/talk/JHVF8N/), given by Martin Schwaighofer. He tackled the problem of reproducibly building cryptographically signed executables. Cryptographic signatures of executables are used for providing trust in the source of a program. If the executable has been signed by a party that you trust, you can run it with confidence. But this confidence is based on the fact that only the trusted part has access to the signing keys. Since the signature is embedded into the resulting artifact, it is not reproducible by anyone but the signer, which defeats the main purpose of reproducible builds; third parties need to be able to reproduce the build in order to attest to its integrity. Martin approached this problem by creating two separate derivations (a derivation is a build action in Nix), only one of which is signed, and comparing the differences between the two. The reproducibility of the unsigned derivation allows for verification of the signed one.
+
+Reproducibility is not all about software security. Martin Kowalewski of Stockholm University gave [a talk](https://talks.nixcon.org/nixcon-2022/talk/MYHSKT/) about how he uses Nix as a part of his research in computational chemistry. The high performance computing software that underpins his research is often made in house at the academic institutions that uses them. The programs tend to grow organically over many years, as demands change. Often, however, maintainability is not the primary concern of the programmers. They are chemists that need to run an experiment, not software engineers. Since the code tends to be quite low level, reproducibility quickly becomes a concern. The output may depend subtly on the exact versions of libraries for scientific computations, and sometimes even on the hardware on which it is executed. Prof. Kowalewski explained that Nix allowed him to eliminate variable factors, which increased the reproducibility and maintainability of his software. In other words, Nix helped scientific experiments to be reproducible.
+
+Not all talks were given by men named Martin. These were just a few highlights. A complete schedule with all talks and speakers can be found [here](https://talks.nixcon.org/nixcon-2022/schedule/).
+
```

Comment:
Could you finish with the two/three main take aways (bullet points)

And maybe a short speculative paragraph on what the CHAINS ideas could bring to NIX?